client/src/components/container/admin/UsersPage/index.js

```diff
@@ -0,0 +1,81 @@
+import React,{Component} from "react";
+import ReactHTMLTableToExcel from 'react-html-table-to-excel';
+import Axios from "axios";
+import { Button } from "antd";
+import {LoadingOutlined} from "@ant-design/icons"
+
+import "./style.css"
+
+class UsersPage extends Component {
+    state={
+        users:[],
+        isLoaded:false,
+        error:null
+    }
+    componentDidMount(){
+        const {match:{params:{eventcode}}}=this.props
```

Comment:
you will receive it as `event_code` from the previous route in the params.
```suggestion
        const {match:{params:{event_code}}}=this.props
```